app/views/devise/registrations/edit.html.erb

```diff
@@ -41,3 +41,10 @@
 <p>Unhappy? <%= button_to "Cancel my account", registration_path(resource_name), data: { confirm: "Are you sure?" }, method: :delete %></p>
```

Comment:
rm for now

Reply:
removed:
<p>Unhappy? <%= button_to "Cancel my account", registration_path(resource_name), data: { confirm: "Are you sure?" }, method: :delete %></p>